docs/diagrams/ModelClassDiagram.puml

```diff
@@ -32,25 +32,25 @@ Class Tag
 Class HiddenOutside #FFFFFF
 HiddenOutside ..> Model
 
-AddressBook .up.|> ReadOnlyAddressBook
+RecipeBook .up.|> ReadOnlyRecipeBook
 
 ModelManager .up.|> Model
 Model .right.> ObservableList
-ModelManager o--> "1" AddressBook
+ModelManager o--> "1" RecipeBook
 ModelManager o-left-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
-AddressBook *--> "1" UniquePersonList
-UniquePersonList o--> "*" Person
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
-Person *--> "*" Tag
+RecipeBook *--> "1" UniqueRecipeList
+UniqueRecipeList o--> "*" Recipe
+Recipe *--> "*" Tag
+Recipe *--> TagList
```

Comment:
not sure about this again, but it looks like the original statements did not have this taglist thing?